Assets/Scripts/PickupScript.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+
+[RequireComponent(typeof(AudioSource))]
+public class PickupScript : MonoBehaviour
+{
+
+	[SerializeField] private AudioClip collectSound;
+	[SerializeField] private GameObject collectEffect;
+    [SerializeField] private CollectibleTypes collectibleType;
+    private Player _player;
+	private GameManager _manager;
+	private float _rotationSpeed;
+	private AudioSource _source;
+	private bool delete;
```

Comment:
Name 'delete' does not match rule 'Instance fields (private)'. Suggested name is '_delete'.